### ISSUE TITLE
Create logo for fluentnode

### ISSUE THREAD

**Issue by DinisCruz** [o2platform/fluentnode#81]
@luchia following on your kind offer to help, when you have some cycles, it would be great if you had a go at creating a logo for fluentnode (this project)

**Comment by luchia** [comment]
Bookmarked :)

**Comment by luchia** [comment]
Hey @DinisCruz, what do you think of this concept?

![logomark](https://cloud.githubusercontent.com/assets/4547737/7695718/20b96982-fe37-11e4-8d8c-854ae6ed0a58.png)

**Comment by DinisCruz** [comment]
That is pretty cool :)

Would you also have a version in colour?

**Comment by luchia** [comment]
Yes :) I did a few different ideas but I think this one came out best. I wanted to show you and get feedback before I put more time in. Will come back to you soon with more.

**Comment by DinisCruz** [comment]
I like it :)
